prov/efa/src/rxr/rxr_pkt_type_base.c

```diff
@@ -260,8 +267,28 @@ ssize_t rxr_pkt_copy_data_to_rx_entry(struct rxr_ep *ep,
 
 	pkt_entry->x_entry = rx_entry;
 
-	if (data_size == 0)
-		goto out;
+	/*
+	 * Under 3 rare situations, this function does not perform the copy
+	 * action, but still consider data is copied:
+	 *
+	 * 1. application cancelled the receive, thus the receive buffer is not
+	 *    available to copy. In the case, this function is still called because
+	 *    receiving end point did not notify the sender about the cancelation,
+	 *    thus sender will keep sending.
```

Comment:
I would rephrase these to  
> this function is still called, otherwise the sender will keep sending as the receiving end point did not notify the sender about the cancelation

Similar to the scenario 2.

Reply:
Sure,  but I would change "otherwise" to "because".

Reply:
In that case let's say
> because the sender will keep sending if the receiving end point did not notify the sender about the cancelation

Reply:
It cannot be "if" because receiver for sure did not notify the sender about the cancellation.

Reply:
I see. Sounds good.